dev/MRTCore/mrt/mrm/include/mrm/MrmEnvironment.h

```diff
@@ -770,12 +772,12 @@ class EnvironmentMapping : DefObject
         _In_reads_bytes_(dataSizeInBytes) const void* rawData,
         _In_ size_t dataSizeInBytes);
 
-    MRMFILE_ENVIRONMENT_VERSION_INFO m_version;
-    const MRMFILE_ENVIRONMENT_MAPPING_HEADER* m_header;
+    MRMFILE_ENVIRONMENT_VERSION_INFO m_version = {0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
If this is a POD (plain-old-data) type, you can probably just do `MRMFILE_ENVIRONMENT_VERSION_INFO m_version {};`

Reply:
Fixed!